src-lib/blas.cpp

```diff
@@ -342,6 +355,36 @@ void pow_cpu(int N, float ALPHA, float *X, int INCX, float *Y, int INCY)
 	for(i = 0; i < N; ++i) Y[i*INCY] = pow(X[i*INCX], ALPHA);
 }
 
+#ifdef USE_OPENBLAS
```

Comment:
Note I've implemented this differently in the .h file instead.  Thank you for this enhancement, @suminoshi.